test/higher_order_ops/test_with_effects.py

```diff
@@ -376,7 +376,7 @@ def add_hooks(module, config):
     @skipIfNoDynamoSupport
     def test_effectful_custom_op_with_subclass(self):
         with torch.library._scoped_library("_mylib", "FRAGMENT") as lib:
-            lib.define("foo(Tensor x) -> Tensor")
+            lib.define("zoo(Tensor x) -> Tensor")
```

Comment:
Can you add a test where we have an regular operator whose backward calls an operator that has a side effect?